src/components/table/columns.ts

```diff
@@ -34,6 +39,32 @@ export class ColumnDefinitionFactory {
     }
 }
 
+/**
+ * Formats the header of the column
+ *
+ * @param {Column} column the configuration for the column
+ *
+ * @returns {string | HTMLElement} custom component that renders a column header
+ */
+export const formatHeader = (column: Column) => (): string | HTMLElement => {
+    const element = document.createElement('div');
+    element.innerText = column.title;
+    if (!column.headerComponent) {
+        return;
+    }
+
+    const menu = document.createElement(column.headerComponent.name);
+    menu.setAttribute('class', 'tabulator-header-menu-button');
```

Comment:
What is this class used for? We could probably use a better name for it as well. What do you think? :slightly_smiling_face:

Reply:
This would be good to have it for styling

Reply:
But who is using it? We do not know what component the user is adding, and they can style their component on their own :thinking: